Lib/Conf.py

```diff
@@ -167,7 +170,8 @@ def SSLProrocol():
 
 
 def getStaticDirs():
-  return getCSValue("StaticDirs", [])
+  staticDirs = getCSValue("DefaultStaticDirs") or ['defaults', 'demo']
```

Comment:
Why we need DefaultStaticDirs and StaticDirs in CS? What is the different?

Reply:
In Core/HandlerMgr.py is added some standard paths for static files:
`68 | statDirectories = ['defaults', 'demo'] + Conf.getStaticDirs()`

But for example this paths don't need for http endpoint of OAuthService. So, is added posibility to configure this default paths.

Reply:
but you break the compatibility now. Imagine if someone have the following setup:
DefaultStaticDirs = 'a','b' and also using defaults as well.
In this modification only 'a' and 'b' directories will be served... LHCb is not the case but probably others experiments are using some of the directory. Please keep the backward compatibility.

Reply:
In this case a someone dont want to use defaults ['defaults', 'demo'] and he replaced it on ['a', 'b'] and after that in result we have:
StaticDirs +  ['a', 'b']
If someone want just add to defaults ['defaults', 'demo'] some statics, he have original option "StaticDirs" is this case we have:
StaticDirs +  ['defaults', 'demo']

And with the last commit:
 if someone do not need any static dirs he just set DefaultStaticDirs to 'None' in this case we have:
StaticDirs +  [ ]

Reply:
fine with me